.github/workflows/ci.yml

```
@@ -292,3 +302,50 @@ jobs:
                 tag: '${{ github.sha }}'
               }
             })
+
+  deploy-on-neotoolsuite-dev:
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    name: Deploy to Neotoolsuite Dev
+    runs-on: ubuntu-latest
```

is this deliberately missing compared to standalone CI?
```suggestion
    runs-on: ubuntu-latest
    timeout-minutes: 5
```